systemtests/tests/py3plug-fd-ldap/CMakeLists.txt

```diff
@@ -35,12 +35,10 @@ set(SYSTEMTEST_LDAP_PASSWORD
     "admin"
     CACHE STRING "LDAP bind password for plugin test"
 )
-check_pymodule_available(2 ldap)
-if(TARGET python-fd
-   AND PYMODULE_2_LDAP_FOUND
-   AND Python2_EXECUTABLE
-   AND SYSTEMTEST_LDAP_ADDRESS
-)
+# check_pymodule_available(2 ldap) if(TARGET python-fd AND PYMODULE_2_LDAP_FOUND
+# AND Python2_EXECUTABLE AND SYSTEMTEST_LDAP_ADDRESS )
+
+if(FALSE) # python2 not supported anymore
```

Comment:
I guess we will have to make sure the testsuite for the new LDAP plugin still works.